pgtap/lineGraph/lineGraph/lineGraph-rows-check.sql

```diff
@@ -0,0 +1,81 @@
+\i setup.sql
+
+SELECT plan(20);
+
+SET extra_float_digits = -3;
+
+-- Check whether the same set of rows is always returned
+
+PREPARE expectedOutputDirected AS
+SELECT * FROM pgr_lineGraph(
+    'SELECT id, source, target, cost, reverse_cost
+    FROM edge_table
+    ORDER BY id'
+);
+
+PREPARE descendingOrderDirected AS
+SELECT * FROM pgr_lineGraph(
+    'SELECT id, source, target, cost, reverse_cost
+    FROM edge_table
+    ORDER BY id DESC'
+);
+
+PREPARE randomOrderDirected AS
+SELECT * FROM pgr_lineGraph(
+    'SELECT id, source, target, cost, reverse_cost
+    FROM edge_table
+    ORDER BY RANDOM()'
+);
+
+PREPARE expectedOutputUndirected AS
+SELECT * FROM pgr_lineGraph(
+    'SELECT id, source, target, cost, reverse_cost
+    FROM edge_table
+    ORDER BY id',
+    directed => false
+);
+
+PREPARE descendingOrderUndirected AS
+SELECT * FROM pgr_lineGraph(
+    'SELECT id, source, target, cost, reverse_cost
+    FROM edge_table
+    ORDER BY id DESC',
+    directed => false
+);
+
+PREPARE randomOrderUndirected AS
+SELECT * FROM pgr_lineGraph(
+    'SELECT id, source, target, cost, reverse_cost
+    FROM edge_table
+    ORDER BY RANDOM()',
+    directed => false
+);
+
+SELECT set_eq('expectedOutputDirected', 'descendingOrderDirected', '1: Should return same set of rows');
```

Comment:
./../pgtap/lineGraph/lineGraph/lineGraph-rows-check.sql                                             (Wstat: 0 Tests: 20 Failed: 10)
  Failed tests:  1-5, 11-15
Mark the failed tests with `todo`